### PR TITLE
Fixed links in new tabs

### LINKED ISSUE #2347: [Task] Modify content such that links open on different tabs.
**Current Behavior**

The links under the pre-requisites section redirect user away from the Learning path [website](https://layer5.io/learn/learning-paths/mastering-service-meshes-for-developers). 
For reference see this : 
https://user-images.githubusercontent.com/81250703/141525085-eef17e46-6dda-46dd-aead-a6a38d0419f1.mp4

**Desired Situation**

The links should open on a different tab.

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by Sayantan-s**
**Description**

*Fixed links opening in same tabs to different tabs.*

Signed-off-by: [Sayantan Samanta](sssmanta789@gmail.com)

This PR fixes #2347

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — COMMENTED** [review]
@Nikhil-Ladha, can `Link` be enhanced to define `target` and `rel`?

@Sayantan-s, thank you. Please note that we want follower on these internal links, and that we'd like to use `Link` for all internal links.

**Comment by Nikhil-Ladha** [comment]
This PR includes changes that are not related to the issue it is addressing. Please fix that @Sayantan-s

**Comment by Nikhil-Ladha** [comment]
I think `target` and `rel` will also work for the `Link` tag as it simply adds the extra props passed to it, to the `a` tag.
@Sayantan-s you can try it out, and confirm.

**Comment by YashKamboj** [comment]
@Nikhil-Ladha "target" does not work in link , have to use "a" tag 
can take a look in this https://github.com/gatsbyjs/gatsby/issues/13825

**Comment by Nikhil-Ladha** [comment]
Ok, thanks for the link 👍🏼.
@Sayantan-s let's continue with the `a` tag for this case.

File: src/components/Learn-Components/Setup-Pre-Requisites/index.js [hunk 1]
```diff
@@ -13,33 +12,39 @@ const SetupPreReq = () => (
       <p>In this learning path, we will be using Meshery as the management plane to manage the service meshes. Meshery is the service mesh management plane which offers lifecycle, configuration, and performance management of service meshes and their workloads.</p>
     </div>
     <div className="container-card" id="Set up">
-      <Link to="/service-mesh-management/meshery">
+      <a href="/service-mesh-management/meshery" 
+        target = "_blank" 
+        rel = "noopener noreferrer">
```
Comment (leecalcote) [inline src/components/Learn-Components/Setup-Pre-Requisites/index.js:17]: The `rel` attribute needs to be removed.

Comment (Sayantan-s) [reply src/components/Learn-Components/Setup-Pre-Requisites/index.js:17]: I'll remove it by today. @leecalcote

**Comment by leecalcote** [comment]
@Sayantan-s do you have time to incorporate the remainder of the feedback here?

**Comment by leecalcote** [comment]
Final follow up... @Sayantan-s, you there?

File: src/components/Learn-Components/Setup-Pre-Requisites/index.js (hunk 1 above)
Comment (Sayantan-s) [reply src/components/Learn-Components/Setup-Pre-Requisites/index.js:17]: Hey man, I'll remove it by today.
Sorry for the delay.

**Comment by leecalcote** [comment]
@warunicorn19, do you mind finalizing this abandoned PR?

**Comment by warunicorn19** [comment]
Sure i'll put in a fix.

File: src/components/Learn-Components/Setup-Pre-Requisites/index.js (hunk 1 above)
Comment (warunicorn19) [inline src/components/Learn-Components/Setup-Pre-Requisites/index.js:17]:
```suggestion
        >
```

File: src/components/Learn-Components/Setup-Pre-Requisites/index.js [hunk 2]
```diff
@@ -13,33 +12,39 @@ const SetupPreReq = () => (
...
             <BsTerminalFill />
           </div>
           <p>Meshery provides you with a clean, robust, streamlined command-line interface to manage your service meshes: 'mesheryctl'.</p>
         </div>
-      </Link>
-      <Link to="/service-mesh-management/meshery/getting-started" id="Run Meshery">
+      </a>
+      <a href="/service-mesh-management/meshery/getting-started" id="Run Meshery"
+        target = "_blank" 
+        rel = "noopener noreferrer">
```
Comment (warunicorn19) [inline src/components/Learn-Components/Setup-Pre-Requisites/index.js:28]:
```suggestion
        >
```

File: src/components/Learn-Components/Setup-Pre-Requisites/index.js [hunk 3]
```diff
@@ -13,33 +12,39 @@ const SetupPreReq = () => (
...
             <img src={Meshery} />
           </div>
           <p>With 'mesheryctl', not only you can manage your service meshes, but also manage their workloads, mesure their performance, verify conformance to service mesh standards.</p>
         </div>
-      </Link>
-      <Link to="/service-mesh-management/meshery/operating-service-meshes" id="Manage Meshery">
+      </a>
+      <a href="/service-mesh-management/meshery/operating-service-meshes" id="Manage Meshery"
+        target = "_blank" 
+        rel = "noopener noreferrer">
```
Comment (warunicorn19) [inline src/components/Learn-Components/Setup-Pre-Requisites/index.js:39]:
```suggestion
       >
```

**Comment by l5io** [comment]
🚀 Preview for commit af28ec212e48d1b9e215786c58a0b6a1bbf612ba at: https://61bc00fe22128164226e7407--layer5ng.netlify.app

**Review by warunicorn19 — APPROVED** [review]

**Pull request merged**